docs/developers/add-new-bidder.md

```diff
@@ -33,6 +33,13 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- category in primary ad server format in `bid.bidVideo.PrimaryCategory`, string value, or single IAB category for each bid in `TypedBid.bid.Cat` field, singe string value in string list, please refer to this convenience link (not the final official definition): [IAB categories](https://adtagmacros.com/list-of-iab-categories-for-advertisement/)
```

Comment:
Veronica, I believe there is a typo in the word "singe" I bet you meant "single". I find the sentence a little confusing, can we please rephrase it? Did you mean that the bidder will return a `string` with either the value found in `bid.bidVideo.PrimaryCategory` or the one in `TypedBid.bid.Cat`?

Reply:
@guscarreon  oh yes, sure, will fix and rephrase. Bidder must return one of those: `bid.bidVideo.PrimaryCategory`, string, or `TypedBid.bid.Cat` just first string in string array.

Reply:
Updated